norent/tests/test_schema.py

```diff
@@ -281,7 +281,7 @@ def test_full_name_mutation_updates_session_if_logged_out(graphql_client):
     output = graphql_client.execute(
         """
         mutation {
-          output: norentFullName(input: {
```

Comment:
If you _really_ want to make sure that the deprecated endpoint still works, you could make a copy of these tests that use the deprecated mutation (i.e., that look exactly like the tests without the changes you've made here).  There are certainly valid arguments that doing so could be "going overboard" but it's totally up to you.

Reply:
done